scanners/amass/templates/amass-scan-type.yaml

```diff
@@ -1,14 +1,13 @@
 # SPDX-FileCopyrightText: 2021 iteratec GmbH
 #
 # SPDX-License-Identifier: Apache-2.0
+
 apiVersion: "execution.securecodebox.io/v1"
 kind: ScanType
 metadata:
   name: "amass{{ .Values.scanner.nameAppend | default ""}}"
 spec:
   extractResults:
-    # amass saves it's result now in json but in "json lines" format
```

Comment:
Should this comment really be removed?
I've added it because I would have found it helpful to know that jsonl is not a typo